src/main/java/betterquesting/client/QuestNotification.java

```diff
@@ -76,19 +76,24 @@ public void onDrawScreen(RenderGameOverlayEvent.Post event) {
             alpha = MathHelper.clamp_float(alpha, 0.02F, 1F);
             final int color = new Color(1F, 1F, 1F, alpha).getRGB();
 
+            GL11.glColor4f(1F, 1F, 1F, alpha);
+
             if (notice.icon != null && alpha > 0.2F) {
                 RenderUtils.RenderItemStack(mc, notice.icon, width / 2 - 8, height / 4 - 20, "", color);
             }
 
-            String tmp = EnumChatFormatting.UNDERLINE + "" + EnumChatFormatting.BOLD + QuestTranslation.translate(notice.mainTxt);
-            int txtW = RenderUtils.getStringWidth(tmp, mc.fontRenderer);
-            mc.fontRenderer.drawString(tmp, width / 2 - txtW / 2, height / 4, color, true);
+            GL11.glEnable(GL11.GL_BLEND);
+            GL11.glBlendFunc(GL11.GL_SRC_ALPHA, GL11.GL_ONE_MINUS_SRC_ALPHA);
 
-            tmp = QuestTranslation.translate(notice.subTxt);
-            txtW = RenderUtils.getStringWidth(tmp, mc.fontRenderer);
-            mc.fontRenderer.drawString(tmp, width / 2 - txtW / 2, height / 4 + 12, color, true);
+            if(alpha > 0.2F){
+                String tmp = EnumChatFormatting.UNDERLINE + "" + EnumChatFormatting.BOLD + QuestTranslation.translate(notice.mainTxt);
+                int txtW = RenderUtils.getStringWidth(tmp, mc.fontRenderer);
+                mc.fontRenderer.drawString(tmp, width / 2 - txtW / 2, height / 4, color, true);
 
-            GL11.glColor4f(1F, 1F, 1F, 1F);
```

Comment:
you shouldn't delete this since it's going to leave the alpha != 1F after exiting this method